shrine/lib/category_menu_page.dart

```diff
@@ -75,9 +75,8 @@ class CategoryMenuPage extends StatelessWidget {
         padding: EdgeInsets.only(top: 40.0),
         color: kShrinePink100,
         child: ListView(
-            children: _categories
-                .map((Category c) => _buildCategory(c, context))
-                .toList()),
+            children:
+                _categories.map((c) => _buildCategory(c, context)).toList()),
```

Comment:
nit: throw a penultimate comma in here for the formatter: `...toList(),),`

Reply:
done